cluster/addons/calico-policy-controller/cni-network-configmap.yaml

```diff
@@ -0,0 +1,23 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
```

Comment:
Unfortunately no :(

I removed it for now to get unblocked and just inlined the config, but would be happy to try some other ideas if you have them!